__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require("supertest");
+const app = require("../app/app");
+
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe('/api/categories', () => {
+    test('GET should respond with a status:200 and an object', () => {
```

Comment:
All 3 /api/categories tests could all be the same 1 single test here with multiple assertions, no need to resubmit it but i would refactor it after to just have 1 /api/categories test and do all the expect()'s in 1